OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -826,4 +821,4 @@
     <Message Text="DEBUG TargetName:             $(TargetName)"/>
     -->
   </Target>
-</Project>
+</Project>
```

Comment:
It was done by VS 2015. I will revert it, but if it is done by VS each time, it will be always manual action.